lms/djangoapps/django_comment_client/base/views.py

```diff
@@ -177,6 +177,14 @@ def update_thread(request, course_id, thread_id):
     if "commentable_id" in request.POST:
         course = get_course_with_access(request.user, 'load', course_key)
         commentable_ids = get_discussion_categories_ids(course)
+        # The commentable ID on thread creation may not be available in the
+        # topic menu, such as when the thread is concerning the current
+        # unit in context. In this case the topic menu will be blank, but the
+        # old commentable_id will be sent.
```

Comment:
@antoviaque Possibly. I'll look at doing this after I finish the test, since the test is a UI one that should work either way.